doc/stats.rst

```diff
@@ -0,0 +1,98 @@
+`Pairtools stats` as a source of quality control metrics
+===========================================
+
+Overview
+--------
+
+`pairtools stats` produces a human-readable nested dictionary of statistics stored in
+a YAML file or a tab-separated text table (specified through the parameters).
+
+- **Summary statistics** include the total number of pairs based on their type, and the number of trans contacts.
```

Comment:
Summaries include things like fraction cis, fraction duplicates etc, so they are all derived from the rest. Not sure how to phrase it nicely.